index.html

```diff
@@ -9,6 +9,23 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="book-container column">
+    <div class="row">
+        <h1 class="book-title">Título del libro</h1>
+        <p class="book-gender">(género)</p>
+    </div>
+    <div class="row">
+        <h3 class="book-author text-1">Autor del libro:</h3>
+        <p class="book-description text-1-description">Nombre del autor del libro</p>
+    </div>
+    <div class="row">
+        <h3 class="book-editorial text-1">Editorial:</h3>
+        <p class="book-description text-1-description">Nombre de la editorial</p>
+    </div>
+    <div class="row">
+        <h3 class="book-year text-1">Año de publicación:</h3>
+        <p class="book-description text-1-description">Año de publicación</p>
```

Comment:
Same. 

Also @pabloqv10. I think you 4 spaces of indent here. We use 2 in general

Reply:
I use 2 spaces on indent, it looks like this on github